<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuarantinePage</class>
 <widget class="QWidget" name="QuarantinePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="backButton">
     <property name="font">
      <font>
       <family>Serif</family>
      </font>
     </property>
     <property name="styleSheet">
      <string>
       QPushButton {
           background-color: #e0e0e0;
           border: 1px solid #c0c0c0;
           border-radius: 3px;
           padding: 5px 15px;
       }
       QPushButton:hover {
           background-color: #d0d0d0;
       }
      </string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <family>Serif</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string>
       QLabel {
           font-size: 18px;
           color: #cc0000;
           padding: 10px;
       }
      </string>
     </property>
     <property name="text">
      <string>Quarantined Files</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="fileList">
     <property name="styleSheet">
      <string>
       QListWidget {
           background-color: #f0f0f0;
           border: 1px solid #c0c0c0;
           border-radius: 4px;
       }
       QListWidget::item {
           padding: 8px;
           border-bottom: 1px solid #e0e0e0;
       }
       QListWidget::item:selected {
           background-color: #cc0000;
           color: white;
       }
      </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="deleteButton">
       <property name="font">
        <font>
         <family>Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton {
             background-color: #cc0000;
             color: white;
             border: none;
             padding: 8px 16px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #b30000;
         }
         QPushButton:disabled {
             background-color: #cccccc;
         }
        </string>
       </property>
       <property name="text">
        <string>Delete File</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="keepButton">
       <property name="font">
        <font>
         <family>Serif</family>
        </font>
       </property>
       <property name="styleSheet">
        <string>
         QPushButton {
             background-color: #666666;
             color: white;
             border: none;
             padding: 8px 16px;
             border-radius: 4px;
         }
         QPushButton:hover {
             background-color: #555555;
         }
         QPushButton:disabled {
             background-color: #cccccc;
         }
        </string>
       </property>
       <property name="text">
        <string>Keep in Quarantine</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
